tests/Command/CommandTest.php

```diff
@@ -99,4 +111,28 @@ public function setting_unknown_defaults_throws_an_exception()
             'doesnotexist' => '0',
         ]);
     }
+
+    /**
+     * @test
+     */
+    public function arguments_that_do_not_map_to_defaults_are_not_added()
+    {
+        $this->application->command('greet [name]', [new GreetCommand, 'greet']);
```

Comment:
I'm not sure I understand what this test do? There is no assertion and the command is not run.